src/extension/atlas/atlas_unwrap.cpp

```diff
@@ -0,0 +1,197 @@
+/**************************************************************************/
+/*  register_types.cpp                                                    */
```

Comment:
Are these the exact copyright headers? Because I think it is supposed to have the file name here which doesn't match the name of the file (so I'm just checking that you didn't accidentally grab the wrong headers).

Reply:
Those are the right copyright headers, I just decided to change the file name.

Here's the original file:
https://github.com/godotengine/godot/blob/master/modules/xatlas_unwrap/register_types.cpp

Reply:
Thinking about it, this license probably doesn't apply to atlas_unwrap.hpp, considering that no parts of it were copied from Godot

Reply:
I see, it is certainly common that a bunch of godot files are called "register_types".